code/modules/mob/living/simple_animal/abnormality/he/rudolta.dm

```diff
@@ -1,7 +1,7 @@
 //ho ho hoe -gail
 /mob/living/simple_animal/hostile/abnormality/rudolta
 	name = "Rudolta of the Sleigh"
-	desc = "An abnormality consisting of three parts: A hornless, disfigured reindeer, 'Santa' and a sleigh. \
+	desc = "An abnormality consisting of three parts: A hornless, disfigured reindeer, \"Santa\" and a sleigh. \
```

Comment:
Is that how you do "" in html text? i assumed it was impossible.

Reply:
yeah the " \ " character will make any character thats in front of it automatically count as a normal letter instead of a special one.
Thats how you get "" inside of descriptions